index.js

```diff
@@ -1,18 +1,9 @@
-let elements = document.getElementsByTagName('*');
-
-for (let i = 0; i < elements.length; i++) {
-    let element = elements[i]
-
-    for (let j = 0; j < element.childNodes.length; j++) {
-        let node = element.childNodes[j];
-
-        if (node.nodeType === 3) {
-            let text = node.nodeValue;
-            let replaced = text.replace('69', 'Nice 😎');
-
-            if (replaced !== text) {
-                node.nodeValue = replaced;
-            }
+const elements = document.getElementsByTagName('*');
+const elementsArray = Array.from(elements);
+elementsArray.forEach(element => {
+    element.childNodes.forEach(node => {
+        if (node.nodeType === 3 && node.nodeValue.includes('69')) {
+            element.innerHTML = node.nodeValue = node.nodeValue.replace('69', '69 <-- Now <strong>That\'s</strong> Nice 😎');
```

Comment:
How does this affect elements that just mention 69 in a large block of text? (Testing currently)

Reply:
Never mind, the screenshot in Telegram clears this up. Attaching for posterity.

![IMAGE 2020-01-02 12:44:25](https://user-images.githubusercontent.com/13542610 /71688707-9b5d1a80-2d5d-11ea-9703-8f73daa59735.jpg)

Reply:
which part are you referring to?